sdks/wasm/packager.cs

```diff
@@ -242,6 +253,10 @@ enum CopyType
 		if (sdkdir != null) {
 			framework_prefix = tool_prefix; //all framework assemblies are currently side built to packager.exe
 			bcl_prefix = Path.Combine (sdkdir, "wasm-bcl/wasm");
+		} else if (Directory.Exists (Path.Combine (tool_prefix, "framework"))) {
+			framework_prefix = Path.Combine (tool_prefix, "framework"); // Running from the build artifacts layout
+			bcl_prefix = Path.Combine (sdkdir, "bcl");
+			sdkdir = tool_prefix;
```

Comment:
sdkdir is null here.

Reply:
My linux only setup is causing me some testing trouble :) silly merge error, thanks!